spec/unit/provider/user/linux_spec.rb

```diff
@@ -120,4 +120,49 @@
       expect( provider.universal_options ).to eql(["-f", 90])
     end
   end
+
+  describe "compare_user" do
```

Comment:
```suggestion
  describe "compare_user_linux" do
```
Suggesting denoting os since there are more than 1 compare_user methods.